modules/core/src/system.cpp

```diff
@@ -508,6 +508,11 @@ int64 getCPUTickCount(void)
 
 #endif
 
+std::ostream& operator << (std::ostream& out, const TickMeter& tm)
```

Comment:
@alalek i tried to move this part in utility.hpp but get errors. what do you suggest?